src/components/checks/create/(create-question)/CreateQuestionDialog.tsx

```
@@ -18,62 +27,35 @@ export default function CreateQuestionDialog({ children, initialValues }: { chil
         return {
-          ...baseValues,
-          // question: 'Describe the essential parts of a computer.',
-          type,
-          expectation: '',
+          ...instantiateOpenQuestion(),
+          question: '',
+          points: 1,
         }
 
       case 'drag-drop':
+        const dragQuestion = instantiateDragDropQuestion()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)